### PR TITLE
Add stdin username/password entry

### PR DESCRIPTION
**Pull request by Kimundi**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -319,6 +320,17 @@ dependencies = [
 ]
 
 [[package]]
+name = "rpassword"
```
Comment (Diggsey) [inline Cargo.lock:323]: Is this fully portable?

File: src/main.rs [hunk 2]
```diff
@@ -77,8 +78,24 @@ fn color_to_index(color: &[u8]) -> u8 {
 
 fn main() {
     let mut args = env::args().skip(1);
-    let username = args.next().expect("<username> argument");
-    let password = args.next().expect("<password> argument");
+    let mut username = args.next();
```
Comment (Diggsey) [inline src/main.rs]: Is `Args` fused?

File: Cargo.lock (hunk 1 above)
Comment (retep998) [reply Cargo.lock:323]: Windows and whatever termios supports.

Comment (Kimundi) [reply Cargo.lock:323]: Seems to have passing build badges for travis and windows

File: src/main.rs (hunk 2 above)
Comment (Kimundi) [reply src/main.rs]: No idea, will ensure it.

**Pull request merged**